BUILD

```diff
@@ -8,7 +8,7 @@ filegroup(
 genrule(
     name = "gen-std-jsonnet-h",
     srcs = ["stdlib/std.jsonnet"],
-    outs = ["core/std.jsonnet.h"],
+    outs = ["std.jsonnet.h"],
```

Comment:
Ok as long as it works, no need for it to be the same location as the makefile for example.